scalability_and_performance/scaling-worker-latency-profiles.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+:context: scaling-worker-latency-profiles
+[id="scaling-worker-latency-profiles"]
+= Improving cluster stability in high latency environments using worker latency profiles  
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+
+include::snippets/worker-latency-profile-intro.adoc[]
+
+These worker latency profiles are three sets of parameters that are pre-defined with carefully tuned values that control the reaction of the cluster to latency issues  without your needing to determine the best values manually.  
```

Comment:
Another occurrence of 'cluster to latency'. This might be moot, depending on the previous comment.